generate_crowdflower_interface_template.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import csv
+import re
+
+# crowdflower interface template header
+TEMPLATE_HEADER = '''<!-- BEGIN sentence -->
+<h2 style="text-align:center">{{lu}}</h2>
+<br />
+<blockquote style="text-align:center">{{sentence}}</blockquote>
+<!-- END sentence -->
+
+<hr />
+'''
+
+# fe_name(question) block template
+FE_NAME_TEMPLATE = '''
+<!-- BEGIN question %(question_num)d -->
+{%% if %(fe_name_field)s != 'No data available' %%}
+<div class="row">
+  <div class="span5">
+    <cml:radios label="{{%(fe_name_field)s}}" class="rando" name="{{%(fe_name_field)s}}" validates="required" gold="true">
+          <cml:radio label="Nessuno"></cml:radio>%(fe_blocks)s</cml:radios>
+  </div>
+
+  {%% if %(type_predicates)s %%}
+  <div class="span5">
+    <div class="alert alert-info">
+      <h3 style="text-align:center"><i class="icon-info-sign"></i> <span style="color:black">SUGGERIMENTI</span><br />{{fe_name0}} potrebbe essere un:</h3>
+      <ul class="inline">%(type_blocks)s</ul>
+    </div>
+  </div>
+  {%% endif %%}
+
+</div>
+<hr />
+{%% endif %%}
+<!-- END question %(question_num)d -->
+'''
+
+# fe block template
+FE_TEMPLATE = '''
+          {%% if %(fe_field)s != 'No data available' %%}
+          <cml:radio label="{{%(fe_field)s}}"></cml:radio>
+          {%% endif %%}'''
+
+# type block template
+TYPE_TEMPLATE = '''
+        {%% if %(type_field)s != 'No data available' %%}
+        <li>{{%(type_field)s}}</li>
+        {%% endif %%}'''
+
+def generate_crowdflower_interface_template(sheet_file_name, template=None):
```

Comment:
Please rename the 2 function parameters for better auto-documentation:
`sheet_file_name` to `input_csv`
`template` to `output_html`